spec/services/phone_number_capabilities_spec.rb

```diff
@@ -69,10 +69,10 @@
       expect(locality).to eq('Bermuda')
     end
 
-    context 'phonelib returns nil' do
-      it 'returns nil' do
+    context 'phonelib returns default country' do
+      it 'returns the default country' do
         locality = PhoneNumberCapabilities.new('703-555-1212').unsupported_location
-        expect(locality).to be_nil
+        expect(locality).to eq('United States')
       end
```

Comment:
cc @jmhooper @mitchellhenke can I get some eyes on this?

I *think* it's OK that `unsupported_location` returns US now... possibly even OK to just to remove this spec? We use this method when building an error message, but not to detect whether or not to allow the country